.travis.yml

```diff
@@ -1,18 +1,11 @@
-osx_image: xcode7.1
+osx_image: xcode7.2
 language: objective-c
-before_install:
-  - brew update
-  - brew outdated xctool || brew upgrade xctool
-  - gem install cocoapods
-  - pod install
```

Comment:
the docs seem to imply that they do this implicitly if you have a Podfile checked in. I thought I'd test it in the best way I know how ;-)